node/src/Commands.ts

```diff
@@ -847,3 +847,10 @@ export function createLindex(
 ): redis_request.Command {
     return createCommand(RequestType.Lindex, [key, index.toString()]);
 }
+
+/**
+ * @internal
+ */
+export function createZpopmin(key: string, count?: number): redis_request.Command {
+    return createCommand(RequestType.ZPopMin, [key, count ? count.toString() : "1"]);
```

Comment:
if count is undefined, we don't want to pass "1", but instead to pass only they key. This way, if redis changes its default count value it won't break us